docs/guides/hybrid-graphics.md

```diff
@@ -65,20 +65,64 @@ This has been tested on the MacBookPro16,1 and the MacBookPro15,1. The 15,3 and
         sudo gpu-switch -i
         ```
 
-    4.  Reboot into Linux. Display brightness should be working again if it wasn't, and `glxinfo | grep "OpenGL renderer"` should show an Intel GPU. Running programs with `DRI_PRIME=1` will make them render on your AMDGPU (some things do this automatically). You will get more battery time now as your AMD GPU can be turned off when not needed.
+    4.  Install the apple-gmux driver from [here](https://github.com/Redecorating/apple-gmux-t2).
+
+    5.  Reboot into Linux. Display brightness should be working again if it wasn't, and `glxinfo | grep "OpenGL renderer"` should show an Intel GPU. Running programs with `DRI_PRIME=1` will make them render on your AMDGPU (some things do this automatically). You will get more battery time now as your AMD GPU can be turned off when not needed.
 
 !!! note
     As macOS expects the dedicated GPU to be activated at startup, to avoid various display and GPU related problems (like not outputting anything to the display after sleep), switch to the dedicated GPU using `sudo gpu-switch -d` before booting into MacOS.
     Because of this, if you want to use Linux with the iGPU again, you'll need to re run `sudo gpu-switch -i` on your Linux install and **reboot**.
 
-# Use on Windows
+## MacBookPro16,4
+
+The AMD GPU on MacBookPro16,4 is not yet compatible with Linux. As a workaround :-
+
+### If you are able to edit your kernel command line :-
+
+1. This workaround requires GRUB as the bootloader. So make sure you install GRUB.
+
+2. Edit the command line of your boot and add the `nomodeset` kernel parameter the the command line. This will enable you to access your Linux system in safe graphics.
```

Comment:
instead of nomodeset, we might be able to use this to bind the amd gpu to pcistub `pci_stub.ids=1002:7360:106b:021e`, which hopefully would not impact the intel gpu, so step 7 wouldn't be needed. (and that id should be unique to mbp16,4 so we may be able to make it the default in t2 isos)

Reply:
Have we tested this on a 16,4? Also, are you sure it doesn't affect any other model?

Reply:
I don't have a 16,4 to test so until someone tests it we can stick with nomodeset. It shouldn't impact other models as they have different ID's